packages/popover/src/positionUtils.ts

```diff
@@ -1,20 +1,29 @@
 import { Align, Justify, ElementPosition } from './types';
 
-type ReferencePosition = ElementPosition;
-type ContentPosition = ElementPosition;
+interface ElementViewportPositions {
+  referenceElViewportPos: ElementPosition;
+  contentElViewportPos: ElementPosition;
```

Comment:
optional nit, the namespace already defines what this properties are.

```suggestion
  referenceEl: ElementPosition;
  contentEl: ElementPosition;
```

Ditto for `ElementDocumentPositions`

Reply:
I named them as such because the two interfaces are combined at a point, so having `referenceEl, contentEl` in both wouldn't work.

Reply:
Got it, I dig it.